docs/content/InputField.mdx

```diff
@@ -0,0 +1,192 @@
+      <InputField.Validation validationKey="validName">Valid name</InputField.Validation>
+      <InputField.Caption>With or without "@". For example "monalisa" or "@monalisa"</InputField.Caption>
+    </InputField>
+  )
+}
+
+render(ValidationExample)
+```
+
+## Props
+
+### InputField
+
+| Name             | Type                                                                                                                  |     Default      | Description                                                                                                                                                                                                          |
+| :--------------- | :-------------------------------------------------------------------------------------------------------------------- | :--------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| children\*       | `InputField.Label`, `InputField.Caption`, `InputField.Validation`, `Autocomplete`, `TextInput`, `TextInputWithTokens` |        -         | Required                                                                                                                                                                                                             |
+| disabled         | `boolean`                                                                                                             |      false       | Whether the field is ready for user input                                                                                                                                                                            |
+| id               | `string`                                                                                                              | generated string | The unique identifier for this field. Used to associate the label, validation text, and caption text                                                                                                                 |
+| required         | `boolean`                                                                                                             |      false       | Whether this field must have a value for the user to complete their task                                                                                                                                             |
+| validationMap    | `Record<string, 'error'\|'sucess'>`                                                                                   |        -         | A map of validation statuses and their associated validation keys. When one of the validation keys is passed to the `validationResult` prop, the associated validation message will be rendered in the correct style |
```

Comment:
stray `\` in the union here
![Screenshot 2021-12-15 at 15 20 33](https://user-images.githubusercontent.com/13340707/146213811-6ea05788-ce50-4d7c-8750-05b0a837b63f.png)